app/controllers/sessions_controller.rb

```diff
@@ -9,10 +9,41 @@ def create
     if user.authenticate(params[:password])
       session[:user_id] = user.id
       flash[:success] = "Welcome, #{user.name}! You are now logged in."
-      redirect_to '/'
+
+      route = determine_route(user)
+
+      redirect_to route
     else
       flash[:error] = "Sorry, youre not one of us."
-      render :new
     end
   end
+
+  def determine_route(user)
+    # WHEN YOU ARE CHECKING _DIFFERENT_ VALUES IN YOUR CONDITION
```

Comment:
The commented out code is just other ways of doing the bottom code. Ill put this elsewhere but thought maybe it was cool to compare